firmware/rmcs_board/boards/pro/app/board_app.cpp

```
@@ -201,18 +202,25 @@ void init_user_button_and_switch_pins() {
     kUserHsFsSwitchPin.configure_as_input();
 }
 
+void init_gpio_pins() {}
+
+SDK_DECLARE_EXT_ISR_M(IRQn_GPIO0_A, gpio_a_isr)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If SDK_DECLARE_EXT_ISR_M is a macro then please configure it.

(unknownMacro)